src/components/AddItemForm.js

```diff
@@ -87,19 +87,30 @@ class AddItemForm extends React.Component {
           </label>
           <br />
           <label className="schedule">
```

Comment:
This label element is what's causing multiple button to display as hovered. Changing it to `span` fixes it, but I'm not sure if that's accessible. But if we fix that we can merge it.